akka-docs/rst/java/http/server-side/websocket-support.rst

```diff
@@ -122,5 +122,5 @@ The handling code itself will be the same as with using the low-level API.
 
 See the `full routing example`_.
 
-.. _example: @github@/akka-docs-dev/rst/java/code/docs/http/javadsl/server/WebSocketCoreExample.java
-.. _full routing example: @github@/akka-docs-dev/rst/java/code/docs/http/javadsl/server/WebSocketRoutingExample.java
\ No newline at end of file
+.. _example: @github@/akka-docs/rst/java/code/docs/http/javadsl/server/WebSocketRoutingExample.java
```

Comment:
@johanandren In the issue #19964 description, Daniel mentioned to update 'full routing example' to below link:
 https://github.com/akka/akka/blob/v2.4.2/akka-docs/rst/java/code/docs/http/javadsl/server/WebSocketRoutingExample.java
This is a link to WebSocketRoutingExample. If it is incorrect, should I update it with WebSocketCoreExample and create new pull request?